src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -495,16 +500,20 @@ export default defineComponent({
       }
     }
 
-    .btn-remove-signer {
-      width: 20px !important;
-      margin: -4px -6px -4px 0;
-      transform: rotate(45deg);
-      cursor: pointer;
-      transition: variables.$transition-interactive;
-      color: variables.$color-grey-light;
+    .button-plain {
```

Comment:
```suggestion
    .btn-plain {
```

Reply:
https://github.com/superhero-com/superhero-wallet/blob/10ed6a048007169b8becc7cc0dade6ecc6d6aedd/src/popup/components/buttons/BtnPlain.vue#L5

Reply:
Yeah, but you are setting the class name in the line 67.